opcua/common/subscription.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, server, params, handler):
         self._monitoreditems_map = {}
         self._lock = Lock()
         self.subscription_id = None
+        self.unknown_handler = False
```

Comment:
Is this a bool? Should it be self.has_unknown_handlers?

Reply:
Good catch! updated